Heading/Heading.module.less

```diff
@@ -0,0 +1,86 @@
+// Heading.module.less
+//
+@import "../styles/variables.less";
+@import "../styles/mixins.less";
+@import "../styles/skin.less";
+
+.heading {
+	.applySkins({
+		border-bottom: @agate-heading-border-width solid transparent;
+		margin: 0 @agate-spotlight-outset (1.5 * @agate-spotlight-outset) @agate-spotlight-outset;
+		padding: @agate-heading-padding;
+		min-height: (1.5 * @agate-spotlight-outset);
+		letter-spacing: @agate-heading-letter-spacing;
+		line-height: @agate-heading-line-height;
+		color: @agate-heading-text-color;
+
+		&.showLine {
+			border-bottom-color: @agate-heading-border-color;
```

Comment:
Suggest we respect the `customColor`/`color` prop here as well.

Reply:
If we choose to do this, the `@agate-heading-accent-color` may not be necessary since it would fully overlap in purpose with the border color.

Reply:
I initially agreed with this, but after implementing it, I am having second thoughts. I'm thinking of a scenario where the custom color is set, and the showLine and the decorator are both showing (the showLine line demarcating the content below the title and the decorator adding visual styling to the title text), there would be no way to have the two be different colors since they're both based on the prop.